### DISCUSSION TITLE
Is there a way to enable the toggle switch (position accepted) by default

### DISCUSSION THREAD

**Discussion by Hula1267** [orestbida/cookieconsent#591] [General]
Is there a way to enable the toggle switch (position accepted) by default for all the cookies groups on the settings page, and not only the mandatory coookies?

**Comment by orestbida** [comment]
@Hula1267, this was possible in previous versions, and will be added back in the next release, v2.10.0. Until then, you can find this feature available in the [v3](https://cookieconsent.orestbida.com/).

**Reply by Hula1267** [reply]
Thanks for adding back this essential function.
